poly-synth/src/poly-synth-demo.js

```diff
@@ -22,163 +22,368 @@ class PolySynthDemo {
   /**
    * @constructor
    * @param {AudioContext} context The audio context.
-   */
-  constructor(context) {
-    this.masterGain_ = new GainNode(context, {gain: 0.5});
-    this.polySynth_ = new PolySynth(context);
-    this.polySynth_.output.connect(this.masterGain_)
-        .connect(context.destination);
-
-    // Initialize a QWERTY keyboard defined in qwerty-hancock.min.js.
-    // The key A on a computer keyboard maps onto the note C, and the letters
-    // to the right of A (SDFGHJ) map onto the notes above C in a C major scale
-    // (DEFGAB). The letters WETYU map onto the black keys of a piano.
-    this.keyboard_ = new QwertyHancock(
-        {id: 'keyboard', width: 600, height: 150, octaves: 2});
-
-    this.keyboard_.keyDown = this.keyDown.bind(this);
-    this.keyboard_.keyUp = this.keyUp.bind(this);
-  }
-
-  /**
-   * Initialize GUI components.
    * @param {String} gainADSRId The ID of the container for gain ADSR elements
```

Comment:
You pointed it out in the HTML file. I should have changed its name in all the JS files too.